packages/actor-rdf-resolve-quad-pattern-string-source/lib/ActorRdfResolveQuadPatternStringSource.ts

```diff
@@ -0,0 +1,92 @@
+import type { MediatorRdfParseHandle } from '@comunica/bus-rdf-parse';
+import { getContextSource
+  , ActorRdfResolveQuadPattern } from '@comunica/bus-rdf-resolve-quad-pattern';
+import type { IActionRdfResolveQuadPattern, IActorRdfResolveQuadPatternArgs, IActorRdfResolveQuadPatternOutput,
+  MediatorRdfResolveQuadPattern } from '@comunica/bus-rdf-resolve-quad-pattern';
+import { KeysRdfResolveQuadPattern } from '@comunica/context-entries';
+import type { IActorTest } from '@comunica/core';
+import type { IDataSource, ISerializeDataSource } from '@comunica/types';
+import type * as RDF from '@rdfjs/types';
+import { storeStream } from 'rdf-store-stream';
+const streamifyString = require('streamify-string');
+
+/**
+ * A comunica RDF Resolve Quad Pattern String Source RDF Resolve Quad Pattern Actor.
+ */
+export class ActorRdfResolveQuadPatternStringSource extends ActorRdfResolveQuadPattern {
+  private readonly sourceType = 'stringSource';
+  private readonly mediatorRdfParse: MediatorRdfParseHandle;
+  private readonly mediatorRdfQuadPattern: MediatorRdfResolveQuadPattern;
+
+  public constructor(args: IActorRdfResolveQuadPatternStringSource) {
+    super(args);
+    this.mediatorRdfParse = args.mediatorRdfParse;
```

Comment:
This line and the next can be removed. The Actor class ensures that `args` are applied to `this`.